src/lib/isKeywordColor.js

```diff
@@ -0,0 +1,160 @@
+import assertString from './util/assertString';
+
+// https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#color_keywords
+const keywords = [
```

Comment:
We should probably replace the array with a set. `set.has` is more efficient than `indexOf`